changelog/unreleased/issue-2203

```diff
@@ -0,0 +1,11 @@
+Bugfix: Fix reading passwords from stdin
+
+Passwords for `init` and `key add` can now be read from non-terminal stdin
+without having to pad the input to 1000 characters.
+
+Additionally the password is now only read once from non-terminal stdin.
```

Comment:
I'd also leave the last two paragraphs out of the changelog. Reading the password from stdin for most commands did not work properly (or it was very hard to get it working), so I think this isn't used by anybody. Also, the code is very old and it was only reported recently...